src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerdeletionservice/event/publisher/dto/OffenderNoBookingDeletionResult.kt

```diff
@@ -0,0 +1,44 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerdeletionservice.event.publisher.dto
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.databind.annotation.JsonSerialize
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateSerializer
+import java.time.LocalDate
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+data class OffenderNoBookingDeletionResult(
+  @JsonProperty("batchId")
```

Comment:
these all look pointless `JsonProperty` declarations since the name of the field is the same?

Reply:
In this case, it is more for the purpose of being explicit and consistent when compared to the other sqs messages, some of which have different field names to the desired json